geniza/corpus/models.py

```diff
@@ -1041,12 +1045,12 @@ def _merge_footnotes(self, doc):
                 # if there's a partial match (everything but content)
                 if equiv_fn:
                     # if the new footnote has content, add it
-                    if footnote.content:
+                    if Footnote.DIGITAL_EDITION in footnote.doc_relation:
                         self.footnotes.add(footnote)
                     # if the partial match has no content, remove it
                     # (if it has any content, then it is different from the new one
                     # and should be preserved)
-                    if not equiv_fn.content:
+                    if Footnote.DIGITAL_EDITION not in equiv_fn.doc_relation:
                         self.footnotes.remove(equiv_fn)
 
```

Comment:
do we need this check anymore? or will the existing equivalence check handle this case now that we're not comparing content?

Reply:
I'd actually updated the original check to compare the `content_text` property:

```py
if include_content:
    compare_fields.append("content_text")
```

So maybe the new check should be something like this?
```py
# if there's a partial match (everything but content)
if equiv_fn:
    # if the new footnote has content, add it
    if footnote.content_text:
        self.footnotes.add(footnote)
    # if the partial match has no content, remove it
    # (if it has any content, then it is different from the new one
    # and should be preserved)
    else:
        self.footnotes.remove(equiv_fn)
# if neither an exact or partial match, add the new footnote
else:
    self.footnotes.add(footnote)
```

Or should we get rid of the content checking entirely?

Reply:
There's no point checking content — transcription is linked to document+source, so if there is more than one digital edition for that document and source they would get the same content. 

It would be nice to have validation to prevent more than one digital edition for a given source being added to a document, but I'm not sure where to put it. Also, I think we want digital edition footnotes to be separate from other footnotes from the same source, if there are any. (I have to admit, this is making me question whether it's the right solution. But I don't think we can efficiently query for annotations in bulk every place we need to know if a document has digital editions or not.)